src/atoms/LinkBarSearchField.jsx

```diff
@@ -1,45 +1,62 @@
-import React, { Fragment } from 'react';
+/* eslint-disable jsx-a11y/label-has-for, jsx-a11y/label-has-associated-control */
```

Comment:
These rules are ignored because eslint does not correctly parse `<Input>` as an `<input>` element.

Reply:
why extension is jsx among js?

Reply:
copy/paste error - many shiny components are just ported from soldotno/aurora-frontend/shared-components.

Reply:
oh yes, in Labrador stuff )

Reply:
aurora was the previous react app powering sol.no.